lib/honeybadger.ex

```diff
@@ -257,10 +289,32 @@ defmodule Honeybadger do
       |> Map.put(:breadcrumbs, breadcrumbs)
 
     exception
-    |> Notice.new(metadata_with_breadcrumbs, stacktrace)
+    |> Notice.new(metadata_with_breadcrumbs, stacktrace, fingerprint)
+    |> put_notice_fingerprint()
     |> Client.send_notice()
   end
 
+  def notify(exception, metadata, stacktrace) when is_map(metadata) and is_list(stacktrace) do
```

Comment:
This still needs a spec, and we can add a `@doc deprecated: "Use Honeybadger.notify/2 instead"` to point people in the right direction from the docs.